Python/legacy_tracing.c

```diff
@@ -173,6 +183,16 @@ sys_trace_func3(
     return call_trace_func(self, Py_None);
 }
 
+static PyObject *
+sys_trace_func3_null(
```

Comment:
Likewise, rename this to `sys_trace_return`, and rename `sys_trace_func3` to `sys_trace_throw`?